pyisic/_standards/pkd2007/pkd2007.py

```diff
@@ -0,0 +1,1661 @@
+# -*- coding: utf-8 -*-
+"""`PKD2007 Standard <https://stat.gov.pl/en/metainformation/classifications/#Polish%20Classification%20of%20Activities%20(PKD)>`_.
+"""
+from ...types import Category, Classification, Standard, Standards
+
+PKD2007 = Standard(
```

Comment:
Looks like @csmith-sayari found this in [English](https://github.com/sayari-analytics/sayari-graph/blob/b3e9c5017e4ef5c530ae23fb3371a52e8fd2903a/graph_parsers/POL/biznes/website_dictionary.py#L7), can we see if we can find that mapping?

Reply:
Seeing English [here](https://stat.gov.pl/files/gfx/portalinformacyjny/en/defaultstronaopisowa/3249/1/1/pkd2007_en.pdf)

Reply:
yep, i will switch to that